control_panel_api/tests/test_services.py

```diff
@@ -67,11 +65,12 @@ def test_bucket_delete(self, mock_delete_bucket_policies):
         mock_delete_bucket_policies.assert_called()
 
 
-@override_settings(ENV='test', IAM_ARN_BASE='arn:aws:iam::1337')
 class NamingTestCase(SimpleTestCase):
     def test_policy_name_has_readwrite(self):
-        self.assertEqual('bucketname-readonly', services._policy_name('bucketname', readwrite=False))
-        self.assertEqual('bucketname-readwrite', services._policy_name('bucketname', readwrite=True))
+        self.assertEqual('bucketname-readonly',
+                         services._policy_name('bucketname', readwrite=False))
+        self.assertEqual('bucketname-readwrite',
+                         services._policy_name('bucketname', readwrite=True))
```

Comment:
Even though this indentation style is ok by PEP8, it's better to indent at regular levels. Break the line after the open parenthesis and indent 4 spaces instead.

Reply:
This is auto pycharm consistent pep8 formatting

Reply:
Ps. what do we @xoen @andyhd  think about increasing line length? I find 80 to be quite annoyingly unreadable the way it breaks most lines and restrictive considering modern retina screens/IDEs with auto line wrapping. I prefer 100

https://www.python.org/dev/peps/pep-0008/#maximum-line-length

"Some teams strongly prefer a longer line length. For code maintained exclusively or primarily by a team that can reach agreement on this issue, it is okay to increase the nominal line length from 80 to 100 characters"

Reply:
Regarding the long lines, I prefer 80 chars as I often use vertical split screen to work on different/related files side-by-side.

I mean, my editor still word-wraps them anyway, so maybe not a big deal, but I think with more than 80 columns the word-wrapping would happen quite often. Also, don't know how that would work on @andyhd 's end...just my 2 cents. See what Andy thinks but we can give it a try if you really want.

Reply:
80 chars ☑️

Reply:
This isn't really a line-length thing, though - it's indenting to an arbitrary number of spaces, not necessarily a multiple of 4.